tests/components/flo/test_sensor.py

```diff
@@ -0,0 +1,24 @@
+"""Test Flo by Moen sensor entities."""
+from homeassistant.components.flo.const import DOMAIN as FLO_DOMAIN
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+from homeassistant.setup import async_setup_component
+
+from .common import TEST_PASSWORD, TEST_USER_ID
+
+
+async def test_sensors(hass, config_entry, aioclient_mock_fixture):
+    """Test Flo by Moen sensors."""
+    config_entry.add_to_hass(hass)
+    assert await async_setup_component(
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5985897)